examples/shift-expressions/shift_expressions.bal

```diff
@@ -0,0 +1,42 @@
+import ballerina/io;
+import ballerina/lang.'int;
+
+public function main() {
+    int a = 1;
+    // `<<` performs a left shift. The bits shifted in on the right are `0`.
```

Comment:
```suggestion
    // `<<` performs a left shift. The bits shifted to the right are `0`.
```

Reply:
IMO, saying "bits shifted to the right" kinda implies that the existing bits are shifted to the right. But that is not the case.

The existing bits are shifted to the left, while `0`s are shifted in from the right.